includes/admin/class-admin.php

```diff
@@ -1294,7 +1294,7 @@ public function enqueue_scripts( $force = false ) {
 				return;
 			}
 
-			$wpbdp->assets->enqueue_admin_scripts( $force );
+			$wpbdp->assets->register_installation_resources();
```

Comment:
It looks like this should load the enqueue_admin_scripts. Maybe we can just remove the force?